back/app/services/projects_finder_admin_service.rb

```diff
@@ -103,6 +105,22 @@ def self.sort_alphabetically(scope, params)
     )
   end
 
+  def self.sort_by_participation(scope, params)
+    direction = params[:sort] == 'participation_desc' ? 'DESC' : 'ASC'
+
+    filtered_projects_subquery = scope.select('projects.*')
+    filtered_projects = Project.from(filtered_projects_subquery, :projects)
+
+    participants_subquery = Analytics::FactParticipation
+      .where(dimension_project_id: filtered_projects.select(:id))
```

Comment:
I don't fully understand what was filtered out from the scope in `filtered_projects `? I wonder if `.where(dimension_project: scope)` could work to simplify this part?